workers/social/lib/social/models/computeproviders/softlayer.coffee

```diff
@@ -9,11 +9,13 @@ KodingError       = require '../../error'
 
 module.exports = class Softlayer extends ProviderInterface
 
-  @providerSlug = 'softlayer'
+  @providerSlug  = 'softlayer'
 
   @bootstrapKeys = ['key_id']
 
-  @sensitiveKeys = ['api_key']
+  @secretKeys    = ['api_key']
+
+  @sensitiveKeys = ['ssh_private_key', 'ssh_public_key']
```

Comment:
We could make this global - `ssh_private_key` and `ssh_public_key` are going to be effectively a part of credentials for every provider.